src/main/kotlin/io/github/inductiveautomation/kindling/statistics/categories/DevicesByDriverType.kt

```diff
@@ -0,0 +1,100 @@
+package io.github.inductiveautomation.kindling.statistics.categories
+
+import io.github.inductiveautomation.kindling.statistics.GatewayBackup
+
+@Suppress("unused")
+class DevicesByDriverType(override val gwbk: GatewayBackup) : StatisticCategory() {
+    override val name = "DevicesByDriverType"
+
+    val compactLogix by queryStatistic(" SELECT COUNT(*) FROM DEVICESETTINGS WHERE TYPE = 'CompactLogix'") {
```

Comment:
Maybe add a `queryScalarStatistic`...

Reply:
Thought about this, but wasn't sure how to get the appropriate type generically.

Would it just be something like 
```
query.executeQuery().getObject() as T
```

Reply:
Yeah, `getObject()` should probably be fine. Something something reified generics might help.